hazelcast/src/main/java/com/hazelcast/map/impl/LazyMapEntry.java

```diff
@@ -81,6 +82,15 @@ public void remove() {
         valueData = null;
     }
 
+    /**
+     * Checks for the existence of the entry in the map
+     *
+     * @return true if the entry exists
+     */
+    public boolean exists() {
```

Comment:
Slightly misleading method name too. If I have an instance I expect it `exists`. It may be `empty` or `nullified`, wdyt?